kubetest/e2e.go

```diff
@@ -623,10 +623,35 @@ func kubemarkTest(testArgs []string, dump, numNodes string) error {
 		})
 	}
 
+	if err := control.XmlWrap(&suite, "kubectl version", getKubectlVersion); err != nil {
```

Comment:
Why do you need this again? We're already doing it after creating the case cluster I believe.

Reply:
yep I saw the same entry above, will drop this